profileGenerator.js

```diff
@@ -0,0 +1,31 @@
+import gridGenerator from './gridGenerator.js';
+
+const profileGenerator = (object) => {
+  // Element creation
+  const card = document.createElement('div');
```

Comment:
Good job creating speakers section dynamically. But kindly use semantic article element instead of div.